task/sast-snyk-check/0.4/sast-snyk-check.yaml

```diff
@@ -0,0 +1,306 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.3"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: sast-snyk-check
+spec:
+  description: |-
+    Scans source code for security vulnerabilities, including common issues such as SQL injection, cross-site scripting (XSS), and code injection attacks using Snyk Code, a Static Application Security Testing (SAST) tool.
+
+    Follow the steps given [here](https://konflux-ci.dev/docs/how-tos/testing/build/snyk/) to obtain a snyk-token and to enable the snyk task in a Pipeline.
+
+    The snyk binary used in this Task comes from a container image defined in https://github.com/konflux-ci/konflux-test
+
+    See https://snyk.io/product/snyk-code/ and https://snyk.io/ for more information about the snyk tool.
+  results:
+    - description: Tekton task test output.
+      name: TEST_OUTPUT
+  params:
+    - name: SNYK_SECRET
+      description: Name of secret which contains Snyk token.
+      default: snyk-secret
+    - name: ARGS
+      type: string
+      description: Append arguments.
+      default: ""
+    - description: Image URL.
+      name: image-url
+      type: string
+      default: ""
+      # In a future 0.4 version of the task, drop the default to make this required
```

Comment:
The future 0.4 version is now.  So we should drop the default (and the comment).

Reply:
fixed